_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
+This includes applications (software delivery, operations) and infrastructure (user devices, services, networks).
 
 #### Cultural
 
-- https://cloud.google.com/solutions/devops/devops-culture-learning-culture
-- https://cloud.google.com/solutions/devops/devops-culture-job-satisfaction
-- https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture
+- [https://cloud.google.com/solutions/devops/devops-culture-learning-culture](https://cloud.google.com/solutions/devops/devops-culture-learning-culture)
+- [https://cloud.google.com/solutions/devops/devops-culture-job-satisfaction](https://cloud.google.com/solutions/devops/devops-culture-job-satisfaction)
+- [https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture](https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture)
 
 Employees feel that their work is meaningful and leverages their expertise.
-They have the tools and resources necessary to get their work done.
-These include technical resources, such as access to servers or environments necessary to develop and test, or resources needed to learn new skills.
+They have the tools and resources necessary to get work done.
+These include technical resources, such as access to servers or environments necessary to develop and test, or learning resources to acquire new skills.
 Employees spend more time leveraging their creativity to add business value and less time on repetitive tasks.
+They also have access to a modern workplace that encourages collaboration and fosters a culture of flexibility, mobility and well-being.
 
 Employees feel they can be honest with management without fear of reproach and be confident that things will improve.
-Scope of changes are purposefully small, de-risking failures that enables learning as opposed to over analysing risks of non-compliance.
-We learn from failure, hold blameless postmortems and continuously ask questions such as why did it fail?
+Scope of changes are purposefully small, de-risking failures that enables learning as opposed to overanalyzing risks of non-compliance.
+The organization learns from failure, holds blameless postmortems and continuously asks questions such as "Why did it fail?".
 
-Employees spend 20% of their time at work to focus on automation of current processes, prototyping related technologies or keeping up to date with trends and technologies, related to their work or the broader GC IT direction.
-Said time also includes time for completing courses and attending conferences.
-The budget for learning opportunities have increased every year for the past 5 years.
+Employees spend 20% of their time at work focusing on automation of current processes, prototyping related technologies or keeping up to date with trends and technologies, related to their work or the broader GC IT direction.
+Those with great innovation-related skills spend even more time to these tasks.
```

Comment:
I thought 20% was already asking alot.  But its more of an average..  Maybe we should say 
> Teams spend 20% of their time..

Reply:
Very good point. Adjusted the text. Some employees are more comfortable and skilled to be innovators than others. Assigning the metric to the team leaves more flexibility.